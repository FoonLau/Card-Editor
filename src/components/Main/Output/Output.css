.app-main-output {
  display: flex;
  align-items: center;
  justify-content: center;
}
