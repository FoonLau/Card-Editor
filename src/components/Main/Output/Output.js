import React from 'react';
import { connect } from 'react-redux';
import Item from 'components/Item/Item';

export const Output = props => (
  <div className="app-output">
    <Item {...props} />
  </div>
);

const mapState = ({ edittingItem }) => edittingItem;

export default connect(mapState)(Output);
