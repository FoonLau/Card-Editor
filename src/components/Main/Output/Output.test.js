import React from 'react';
import renderer from 'react-test-renderer';
import { Output } from './Output';

it('should render correctly', () => {
  const tree = renderer.create(<Output />).toJSON();

  expect(tree).toMatchSnapshot();
});
