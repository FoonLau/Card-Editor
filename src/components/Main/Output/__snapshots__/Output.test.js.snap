// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="app-output"
>
  <div
    className="app-item"
    style={
      Object {
        "borderRadius": "undefined%",
      }
    }
  />
</div>
`;
